bindings/rust/extended/s2n-tls/src/cert_chain.rs

```diff
@@ -45,20 +47,50 @@ impl CertificateChainHandle<'_> {
             _lifetime: PhantomData,
         }
     }
+
+    fn internal_context_mut(&mut self) -> Option<&mut InternalContext> {
```

Comment:
I don't love the ambiguity of the name "context" either, but I'm not sure "internal" really improves anything and I think consistency is more important here. All our other structs use just "context" rather than "internal_context".